Phonebook.Frontend/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -82,4 +99,23 @@ export class DashboardComponent implements OnInit {
   public removeFromBookmarkedPersons(person: Person) {
     this.store.dispatch(new ToggleBookmark(person));
   }
+
+  public setDrawer() {
```

Comment:
Please use NGXS as state manager. (https://www.ngxs.io/) 
You can add it to https://github.com/T-Systems-MMS/phonebook/blob/master/Phonebook.Frontend/src/app/shared/states/App.state.ts

Reply:
Do I have to add a service in order to do that?